scripts/commands/DrunCommand.ts

```diff
@@ -0,0 +1,26 @@
+import Command from "./Command.ts";
+
+export default class DrunCommand extends Command {
+	public label: string = "command.drun";
+	public name: string = "drun";
+	public aliases: string[] = [
+		"denorun",
+	];
+	public description: string = "Runs the Netrex automatically project if provided an important map";
+	public usage: string = "{c} [path | mod.ts]";
```

Comment:
usage should be on new lines:
```suggestion
	public usage: string = "{c} [module = mod.ts]\n {c} ./index.ts\n {c} ./mod.ts\n {c}";
```